www/.env.development

```diff
@@ -1 +1,2 @@
 FEATURE_DIRECTION_SWITCHER=true
+SEGMENT_KEY=0123456789
```

Comment:
During development, I think it would make sense to have it be `SEGMENT_KEY=''` so that Segment doesn't get configured locally unless explicitly supplying your own `SEGMENT_KEY` here for testing/QA purposes. I might worry that using an incorrect Segment key will throw errors when the script configures Segment.

Reply:
Right, I leave it empty